ml-proto/spec/check.ml

```diff
@@ -259,6 +259,7 @@ and check_has_memory c at =
 
 and check_memop memop at =
   require (memop.offset >= 0L) at "negative offset";
+  require (memop.offset <= 4294967295L) at "offset too large";
```

Comment:
This constant could use hex form to make it more obvious what value it is: 0xffffffffL.